src/main/java/Crystal.java

```diff
@@ -0,0 +1,298 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import exceptions.InvalidCommandException;
+import exceptions.IncompleteCommandException;
+import exceptions.IllegalCommandException;
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Task;
+import tasks.Todo;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Crystal {
+    public static ArrayList<Task> tasks = new ArrayList<>();
+    private static int taskCount = 0;
+    private static final int TODO_CHAR_COUNT = 4;
+    private static final int DEADLINE_CHAR_COUNT = 8;
+    private static final int EVENT_CHAR_COUNT = 5;
+
+    public static void horizontalLine() {
```

Comment:
Perhaps name your method with a verb instead?
```suggestion
    public static void printHorizontalLine() {
```